CMakeLists.txt

```diff
@@ -1,4 +1,25 @@
 cmake_minimum_required (VERSION 2.8.12)
+
+if(Vita OR VITA)
+
+message("VITASDK Defined!")
```

Comment:
I guess, here are indentions needed :thinking:  but no problem, I'll add that by myself for you :wink:

Reply:
Oops :P